main.go

```diff
@@ -89,16 +94,27 @@ func main() {
 	}
 
 	log.Info("Requesting Access Token from Dataporten")
+	err = nil
+	token := ""
+
+	// Manually fetch token if browser is unavailable from console:
+	if (cluster.ManualInput) {
+		fmt.Println("Open a browser and navigate to " + authURL + "?response_type=token&client_id=" + cluster.ClientID + ", then report the return token below.")
```

Comment:
Would be good to inform user that here they need to copy token from the url itself. The value is in the request param `access_token`.